bop_toolkit_lib/dataset_params.py

```diff
@@ -374,6 +381,22 @@ def get_split_params(datasets_path, dataset_name, split, split_type=None):
             p["azimuth_range"] = None  # Not calculated yet.
             p["elev_range"] = None  # Not calculated yet.
 
+    # HOT3D.
+    elif dataset_name == "hot3d":
+        modalities_have_separate_annotations = True 
+        p["im_modalities"] = ["rgb","gray1","gray2"]
+        p["eval_modality"] = "rgb"
+        p["scene_ids"] = {
+            # "trainariasubsample": [1849, 2102],
+            "trainariasubsample": [1901, 2503],  # TODO: replace with actual test split scenes
+        }[split]
+        # p["im_size"] = (1920, 1080)  # Aria != Quest, not applicable
+
+        if split == "test":
+            p["depth_range"] = None  # Not calculated yet.
```

Comment:
We should at least provide a depth_range, azimuth and elevation can be unconstrained

Reply:
Sure, any guesses for the value or should we compute that from the ground truth annotation somehow?

Reply:
Yes, you can estimate it from the L2 norm over the translation from `cam_t_m2c` in scene_gt.json.

Reply:
we can also do this in another PR, I saw that it's neither done yet for the other H3 datasets.